mmdet/datasets/pipelines/transforms.py

```diff
@@ -1783,8 +1786,8 @@ def _test_aug(self, results):
         h, w, c = img.shape
         results['img_shape'] = img.shape
         if self.test_pad_mode[0] in ['logical_or']:
-            target_h = h | self.test_pad_mode[1]
-            target_w = w | self.test_pad_mode[1]
+            target_h = (h | self.test_pad_mode[1]) + self.test_pad_add_pix
```

Comment:
Is this necessary?

Reply:
I have tested it, if not then resnet18 will reduce 0.2map.

Reply:
Need to add comments for this, indicate what model it affects.